app/classes/site_data.rb

```diff
@@ -142,6 +138,34 @@ class SiteData
     contributing_users: "contribution > 0"
   }.freeze
 
+  # Non-default unified queries for stats for the entire site
+  # rubocop:disable Layout/MultilineMethodCallIndentation
+  # Rubocop 1.30.0 wants to allgn "where" with the open brace on the next line.
+  FIELD_QUERIES = {
+    contributing_users:
+      User.
+        where(User[:contribution].gt(0)),
```

Comment:
My First Arel! 
Once I figured out the syntax, this was trivial thanks to the examples and gems provided by @nimmolo.

Reply:
I think a better way to go here is pure AR, thanks to our recent updates we can use endless ranges:
```ruby
User.
  where(contribution: 0..)
```

Reply:
@nimmolo Thanks for pointing me the right way on endless ranges.
After some experimentation, I realized that `0..` is `>= 0`. https://ruby-doc.org/core-3.1.1/Range.html#class-Range-label-Endless+Ranges
Fortunately, user contribution scores are integral, so we can do
```ruby
User.
  where(contribution: 1..)
```